zer0-os/zOS
# feat: display error dialog if active conversation is not valid

### What does this do?
- wires up displaying the error dialog if the active conversation fails the validation check.

### Why are we making this change?
- to display an error to the user if the room url (active conversation id) is invalid.

### How do I test this?
- visit a url that is either `bad` or does not contain a room id that the user is not a member of > check the error dialog is displayed > when closing the dialog check the first conversation in the list opens.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?


https://github.com/zer0-os/zOS/assets/39112648/6f3e7870-9b11-4c9e-b1f1-6f3443299a74
